pkg/cli/cli.go

```diff
@@ -1064,6 +1073,24 @@ func runDisconnectCommand(flags *ConnectFlags, c *command, args []string, opts *
 	return merr
 }
 
+func runGetConfigCommand(c *cobra.Command, args []string, cfg Config) error {
+	if len(args) == 0 {
+		return errors.New("missing parameter")
+	}
+	if len(args) > 1 {
+		return errors.New("use one parameter at a time")
+	}
+	switch args[0] {
```

Comment:
nit: You could implement this once and never have come back to it again by encoding the config object as JSON and then decoding it using a `map[string]interface{}`.
You'd have to split the "property name" by `'.'` to support inner fields, but that shouldn't be too hard.
We are already planning to support multiple service urls, which means the config structure could change, so don't reply with "YAGNI" :)

Reply:
:) done.